flow-typed/box-ui-elements.js

```diff
@@ -542,33 +542,46 @@ type Translations = {
     onTranslate?: Function
 };
 
-type OpenWithIntegrations = {
-    default_app_integration?: AppIntegrationMiniItem,
+type OpenWithAPI = {
+    default_app_integration?: AppIntegrationAPIMiniItem,
     disabled_reasons?: Array<string>,
     icon?: string,
     is_disabled?: boolean,
-    items: Array<OpenWithIntegrationItem>,
+    items: Array<OpenWithAPIItem>,
     should_show_consent_popup?: boolean
 }
 
-type OpenWithIntegrationItem = {
-    app_integration: AppIntegrationMiniItem | AppIntegrationItem,
+type OpenWithAPIItem = {
+    app_integration: AppIntegrationAPIMiniItem,
     disabled_reasons: Array<string>,
     display_order: number,
     icon: string,
     is_disabled: boolean,
     should_show_consent_popup: boolean
 }
 
-type AppIntegrationMiniItem = {
+type AppIntegrationAPIMiniItem = {
     type: 'app_integration',
     id: string
 }
 
-type AppIntegrationItem = {
+type AppIntegrationAPIItem = AppIntegrationAPIMiniItem & {
     name: string,
     description: string
-} & AppIntegrationMiniItem
+} 
+
+type Integration = {
+    appIntegrationId: string,
+    name: string,
+    description: string,
+    type: 'app_integration',
```

Comment:
Since 'app_integration' is used in more than once place, would there be value moving it to a constant?